packages/rspack-test-tools/tests/configCases/loader/rspack-issue-4672/loader.js

```diff
@@ -4,7 +4,9 @@ module.exports = function () {
 	const callback = this.async();
 
 	this.resolve(__dirname, "./not-exist", (e1, r1) => {
-		assert(e1 instanceof Error);
+		// Note that e1 is not instanceof Error
```

Comment:
I'm not sure if this is expected /cc @h-a-n-a

Reply:
No, it's not 🥲

Reply:
@colinaaa This seems to be a known jest issue https://github.com/jestjs/jest/issues/11808 and it turned out the assertion works outside of jest runner.